From 75ce07184e105d64de41beee17cb1d2798b429de Mon Sep 17 00:00:00 2001
From: Hitoshi Harada <umi.tanuki@gmail.com>
Date: Wed, 20 Feb 2013 00:02:04 -0800
Subject: [PATCH 3/5] Rethink debugger support option in static build.

The remote debugger binds a port which means we cannot run multiple
session simultaneously.  It was a bad choice not to give users to
choose if they want the controversial feature.
---
 Makefile.v8 |    2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile.v8 b/Makefile.v8
index f08c7de..c0234c2 100644
--- a/Makefile.v8
+++ b/Makefile.v8
@@ -14,8 +14,6 @@ AUTOV8_LIB = $(AUTOV8_OUT)/libv8_snapshot.a
 AUTOV8_STATIC_LIBS = libv8_base.a libv8_snapshot.a
 
 SHLIB_LINK += $(addprefix $(AUTOV8_OUT)/, $(AUTOV8_STATIC_LIBS))
-# We are sure the static buid can support debugger
-ENABLE_DEBUGGER_SUPPORT=1
 
 all: $(AUTOV8_LIB)
 
-- 
1.7.10.4

